templates/admin/auth/list.tmpl

```diff
@@ -31,6 +31,7 @@
 							<td><a href="{{AppSubUrl}}/-/admin/auths/{{.ID}}">{{svg "octicon-pencil"}}</a></td>
 						</tr>
 					{{end}}
+					{{template "shared/no_results_found" (dict "rst" .Sources "colspan" 7)}}
```

Comment:
It could be as simple as 

```
{{else}}
    <tr><td class="tw-text-center" colspan="7">{{ctx.Locale.Tr "no_results_found"}}</td></tr>
```

No need the `template dict` trick.

Reply:
Actually, I've taken [your previous advice](https://github.com/go-gitea/gitea/pull/33498#issuecomment-2637008355 ). Instead of simply copy - pasting, I've extracted the common parts into template code. Using templates uniformly makes the code easier to maintain.

Reply:
There is also another problem when coding: over abstraction. You need to balance the trade-off.

Such simple change could be done by `else` , but not use a unclear sub-template.